### ISSUE TITLE
Feature request: variable timetable repeat period

### ISSUE THREAD

**Issue by jabailey314** [IncredibleHannes/TPF2-Timetables#95]
This is an incredible mod which completely changes the game and makes it much more enjoyable. I think the single biggest improvement to this mod would be to allow the user to determine the amount of time after which the timetable repeats itself, which is currently set to 60 minutes. Because of the higher density of services required by the game to meet demand compared with real life, it's quite common to have trains running every 2-5 minutes or so, so I often end up with timetables that repeat themselves every 10 or 15 minutes. In this case, I end up just entering the same timetable 4-6 times within the hour, which can get quite laborious. This could be avoided if there was a field to select the repeat period of the timetable (e.g., "repeat every 15 minutes") so that the user would only need to enter the timetable once. It would also be much easier to edit the timetable subsequently if needed. Thank you!

**Comment by quittung** [comment]
I agree, and I have tried to implement those changes. You can look at the pull request here: #90 

As I understand it, the mod author is currently busy with work, so the changes have not been accepted yet. Until that happens, and if you are comfortable with manually installing TF2 mods, you can try my version here: 
https://github.com/quittung/TPF2-Timetables/tree/recurringDepartures
Click on Code, then Download ZIP
